docs/_documentations/overview.md

```diff
@@ -1,29 +1,83 @@
 ---
 layout: docs
-title: Overview
-description: Overview
-keywords: install, standalone, cloud
-duration: 1 minute
+title: What is Codewind?
+description: What is Codewind?
+keywords: IDE, cloud native, kubernetes, cloud, development, VS Code, Eclipse, Eclipse Che, templates, local, remote, hosted, overview, prerequisites
+duration: 2 minute
 permalink: Overview
 type: document
 parent: root
 ---
 
-# Overview
+# What is Codewind?
 
-You can use codewind to develop and debug your containerized projects from within an IDE.
+Building and Developing a cloud native, containerized application can be challenging. 
 
-Write code, track application and build statuses, view project logs, and run your application. Codewind supports development of Appsody, Go, Java Lagom, Microprofile/Java EE, Node.js, Python, Spring, and Swift containerized projects. In addition, the tools support easily debugging Microprofile/Java EE, Spring, and Node.js applications.
+Firstly, you need to create your own application stack for a containerized microservice that also fits your preferred language and project type.
 
+Then, if you are a software developer, you need to perform many actions as you develop, build, run, and test your code. You need to build images, assess build status, create containers, open application end points, debug, analyse the different logs, assess performance metrics, and re-build the containers with each code change.
 
-# Features
+Or, if you are responsible for defining standards for application and runtime environments such as framework and software levels, you need to implement and maintain your standards across the whole development team, ensuring consistency and reliability. 
+
+Built from the ground up, Codewind, an [open source project](https://github.com/eclipse/codewind), helps you to achieve all of the above by extending your Integrated Development Environment(IDE). Container development flow feels like a traditional application flow and [project templates](mdt-vsc-workingwithtemplates.html) can be used across your development teams effortlessly. Support for rapid iteration of your code is achieved using features including automatic updating of containers with code changes, pre-supplied container-based [project templates](mdt-vsc-workingwithtemplates.html), and a user interface that exposes metrics, logs, and load testing at the click of a button. Your own standardised [project templates](mdt-vsc-workingwithtemplates.html) can be accessed by your developers using their IDE of choice.
```

Comment:
- Even though I think someone with a screen reader could probably understand "above" in this context, the DAP checker tool will still flag "above." Maybe something like..."helps you with many application development tasks"?
- Space needed between "Environment" and "(IDE)".
- American spelling: "standardized"